usaspending_api/awards/models_matviews/vw_search_award.py

```diff
@@ -0,0 +1,88 @@
+from django.contrib.postgres.fields import ArrayField
+from django.contrib.postgres.search import SearchVectorField
+from django.db import models
+
+from usaspending_api.awards.models import Award
+
+
+class ViewSearchAward(models.Model):
```

Comment:
While not a matview, it appears this model could also inherit from the same base class as the matviews re: https://github.com/fedspendingtransparency/usaspending-api/pull/1977/files#r316308611